grafana-plugin/src/pages/schedules_NEW/Schedules.tsx

```diff
@@ -51,7 +52,7 @@ class SchedulesPage extends React.Component<SchedulesPageProps, SchedulesPageSta
     const { store } = this.props;
     this.state = {
       startMoment: getStartOfWeek(store.currentTimezone),
-      filters: { searchTerm: '', status: 'all', type: ScheduleType.API },
+      filters: { searchTerm: '', status: 'all', type: ScheduleType.all },
```

Comment:
`all` is the same if we don't specify any specific type when we do request,
`GET /schedules/?type=1` we want iCal's only
`GET /schedules/ ` we want everything